### ISSUE TITLE
make should use -j $(nproc)

### ISSUE THREAD

**Issue by maci0** [zfsonlinux/zfs#2396]
dkms module compilation takes very long.
the rpm generation could also be faster if the .spec template would use this.

**Comment by behlendorf** [comment]
Closing, won't fix.  The configure step can't be easily done in parallel, the make portion of dkms should be running in parallel.
